api_tests/files/views/test_file_list.py

```diff
@@ -1,138 +1,160 @@
-# -*- coding: utf-8 -*-
-from nose import tools as nt
-
 from framework.auth.core import Auth
 
+import pytest
+
 from api.base.settings.defaults import API_BASE
 from api_tests import utils as api_utils
-
 from tests.base import ApiTestCase
 from osf_tests.factories import (
     ProjectFactory,
     UserFactory,
     AuthUserFactory,
 )
 
-class TestNodeFileList(ApiTestCase):
-
-    def setUp(self):
-        super(TestNodeFileList, self).setUp()
-        self.user = AuthUserFactory()
-        self.node = ProjectFactory(creator=self.user)
-        self.file = api_utils.create_test_file(
-            self.node, self.user, filename='file1')
-        self.deleted_file = api_utils.create_test_file(
-            self.node, self.user, filename='file2')
-        self.deleted_file.delete(user=self.user, save=True)
-
-    def test_does_not_return_trashed_files(self):
-        res = self.app.get(
-            '/{}nodes/{}/files/osfstorage/'.format(API_BASE, self.node._id),
-            auth=self.user.auth
+@pytest.fixture()
+def user():
+    return AuthUserFactory()
+
+@pytest.mark.django_db
+class TestNodeFileList:
+
+    @pytest.fixture()
+    def node(self, user):
+        return ProjectFactory(creator=user)
+
+    @pytest.fixture()
+    def file(self, user, node):
+        return api_utils.create_test_file(
+            node, user, filename='file1')
+
+    @pytest.fixture()
+    def deleted_file(self, user, node):
+        deleted_file = api_utils.create_test_file(
+            node, user, filename='file2')
+        deleted_file.delete(user=user, save=True)
+        return deleted_file
+
+    def test_does_not_return_trashed_files(self, app, user, node, file, deleted_file):
+        res = app.get(
+            '/{}nodes/{}/files/osfstorage/'.format(API_BASE, node._id),
+            auth=user.auth
         )
         data = res.json.get('data')
-        nt.assert_equal(len(data), 1)
-
-class TestFileFiltering(ApiTestCase):
-    def setUp(self):
-        super(TestFileFiltering, self).setUp()
-        self.user = AuthUserFactory()
-        self.node = ProjectFactory(creator=self.user)
-        self.file1 = api_utils.create_test_file(
-            self.node, self.user, filename='file1')
-        self.file2 = api_utils.create_test_file(
-            self.node, self.user, filename='file2')
-        self.file3 = api_utils.create_test_file(
-            self.node, self.user, filename='file3')
-        self.file4 = api_utils.create_test_file(
-            self.node, self.user, filename='file4')
-
-    def test_get_all_files(self):
-        res = self.app.get(
-            '/{}nodes/{}/files/osfstorage/'.format(API_BASE, self.node._id),
-            auth=self.user.auth
+        assert len(data) == 1
+
+@pytest.mark.django_db
+class TestFileFiltering:
+
+    @pytest.fixture()
+    def node(self, user):
+        return ProjectFactory(creator=user)
+
+    @pytest.fixture()
+    def file1(self, user, node):
```

Comment:
Personal / Nit, change to be file_one, file_two, etc. (or a more descriptive name if possible)